src/util/transformThemeValue.js

```diff
@@ -13,15 +13,14 @@ export default function transformThemeValue(themeSection) {
       'transitionTimingFunction',
       'backgroundImage',
       'backgroundSize',
-      'backgroundColor',
```

Comment:
I was having a hard time understanding why this was here. I removed it and it didn't break any tests so that's good but I can't help but feel like I'm missing something.